cli/config.go

```diff
@@ -15,12 +16,13 @@ func getCLICachePath() string {
 	return filepath.Join(viamDotDir, "cached_cli_config.json")
 }
 
-func configFromCache() (*config, error) {
+// ConfigFromCache parses the cached json into a Config.
+func ConfigFromCache() (*Config, error) {
```

Comment:
If the pulling of an auth config from cache is now a feature used by the CLI and `app`, should we move this file to be some utility file elsewhere in the RDK?

Reply:
I think we should consider either moving this out of the cli or plan to restructure the cli (with an internal dir or something) to better fit the needs of being both a library and a cli tool

(Aside: I don't think this is entirely out of place in the `rdk/cli` package -- I think I've seen other OSS projects that export cli related tools from their cli package...)

Reply:
no preference about where it lives; in theory could move to goutils

also I'm just exporting the 'dial' part, not the full reconnection / expiration logic (the consumer for this is a one-off internal tool that will get TLC if folks find it useful), so if we think about the right way:tm: to export this, we should make a decision about how much of the client to export

I did it this way to minimize the size of the diff. happy to also move stuff if you have a better home for it